ananas/src/main/java/iamutkarshtiwari/github/io/ananas/editimage/EditImageActivity.java

```diff
@@ -450,30 +452,7 @@ protected void onDestroy() {
     protected void setLockScreenOrientation(boolean lock) {
         if (Build.VERSION.SDK_INT >= 18) {
             setRequestedOrientation(lock ? ActivityInfo.SCREEN_ORIENTATION_LOCKED : ActivityInfo.SCREEN_ORIENTATION_FULL_SENSOR);
-            return;
         }
-
-        if (lock) {
```

Comment:
ok this will not create a problem for us since we will never run this on lower than 18 but I think that it is wrong to completely remove these.
You could change the pattern with an early return though and put an else on the if in line 451 but please keep this code for lower platforms as it will makes sense if we want to do a PR to contribute back to the open-source project